el.json

```diff
@@ -1,1800 +1,1801 @@
-			"tooltip-link-file": "Link this file",
-			"tooltip-not-created": "Not created yet"
+			"name": "Εξερχόμενοι σύνδεσμοι",
+			"desc": "Προβολή εξερχόμενων συνδέσμων και εντοπισμός μη συνδεδεμένων αναφορών για άλλες σημειώσεις στην τρέχουσα σημείωση.",
+			"action-open": "Άνοιγμα εξερχόμενων συνδέσμων",
+			"action-show": "Προβολή παραθύρου εξερχόμενων συνδέσμων",
+			"action-open-for-current": "Άνοιγμα εξερχόμενων συνδέσμων για το τρέχων αρχείο",
+			"tab-title": "Εξερχόμενοι σύνδεσμοι από {{displayText}}",
+			"label-links": "Σύνδεσμοι",
+			"label-no-links": "Δεν βρέθηκαν σύνδεσμοι.",
+			"label-unlinked-mentions": "Μη συνδεδεμένες αναφορές",
+			"tooltip-link-file": "Σύνδεσμος αυτού του αρχείο",
+			"tooltip-not-created": "Δεν δημιουργήθηκε ακόμη"
 		},
 		"tag-pane": {
-			"name": "Tag pane",
-			"desc": "Displays all your tags and along with their number of occurrences.",
-			"action-show": "Show tag pane",
-			"label-no-tags": "No tags found.",
-			"label-sort-by-name-a-to-z": "Tag name (A to Z)",
-			"label-sort-by-name-z-to-a": "Tag name (Z to A)",
-			"label-sort-by-frequency-high-to-low": "Frequency (high to low)",
-			"label-sort-by-frequency-low-to-high": "Frequency (low to high)",
-			"action-show-nested-tags": "Show nested tags",
-			"action-collapse-all": "Collapse all",
-			"action-expand-all": "Expand all"
+			"name": "Παράθυρο ετικετών",
+			"desc": "Προβολή όλων των ετικετών μαζί με τον αριθμό εμφάνισης τους.",
+			"action-show": "Προβολή παραθύρου ετικετών",
+			"label-no-tags": "Δεν βρέθηκαν ετικέτες.",
+			"label-sort-by-name-a-to-z": "Όνομα ετικέτας (Α στο Ω)",
+			"label-sort-by-name-z-to-a": "Όνομα ετικέτας (Ω στο Α)",
+			"label-sort-by-frequency-high-to-low": "Συχνότητα (φθίνουσα)",
+			"label-sort-by-frequency-low-to-high": "Συχνότητα (αύξουσα)",
+			"action-show-nested-tags": "Προβολή ένθετων ετικετών",
+			"action-collapse-all": "Κλείσιμο όλων",
+			"action-expand-all": "Επέκταση όλων"
 		},
 		"page-preview": {
-			"name": "Page preview",
-			"desc": "Preview a note when hovering an internal link. Ctrl/Cmd + hover in editor mode.",
-			"label-empty-note": "\"{{linktext}}\" is not created yet. Click to create.",
-			"label-source-editor": "Editor",
-			"label-source-preview": "Preview",
-			"label-source-search": "Search, Backlinks, and Outgoing links",
-			"label-require-mod": "Require {{key}} to trigger page preview on hover"
+			"name": "Προεπισκόπισης σελίδας",
+			"desc": "Προεπισκόπιση σημείωσης όταν ο κέρσορας είναι πάνω από εσωτερικό σύνδεσμο. Ctrl/Cmd + κλίκ όταν είναι πάνω ο κέρσορας σε λειτουργία συντάκτη.",
+			"label-empty-note": "\"{{linktext}}\" δεν δημιουργήθηκε. Κάντε κλικ για να δημιουργήσετε.",
+			"label-source-editor": "Συντάκτης",
+			"label-source-preview": "Προεπισκόπηση",
+			"label-source-search": "Αναζήτηση, Backlinks, και εξωτερική σύνδεσμη",
+			"label-require-mod": "Απαιτείται {{key}} για να πυροδοτηθεί η προεπισκόπηση σελίδας"
 		},
 		"starred": {
```

Comment:
This section can get removed, as the starred plugin got replaced with the Bookmarks plugin
<https://forum.obsidian.md/t/obsidian-release-v1-2-7/59004>